source/documentation/common/upgrade/proc-Upgrading_the_Manager_to_4-4.adoc

```diff
@@ -62,10 +63,33 @@ ifdef::local_database_upgrade,remote_database_upgrade[. Log in to the {engine-na
 ----
 # engine-backup --scope=all --mode=backup --file=backup.bck --log=backuplog.log
 ----
-
 . Copy the backup file to a storage device outside of the {virt-product-shortname} environment.
 
 ifdef::SHE_upgrade[]
+
+. Shut down the self-hosted engine.
++
+[options="nowrap" subs="normal"]
+----
+# poweroff
+----
++
+[NOTE]
+====
+Before you shut down the self-hosted engine virtual machine, write down the MAC address of the self-hosted engine network interface if you want to reuse it to deploy the {virt-product-fullname} {engine-name} 4.4.
```

Comment:
```suggestion
If you want to reuse the self-hosted engine virtual machine to deploy the {virt-product-fullname} {engine-name} 4.4, then before you shut it down, note the MAC address of the self-hosted engine network interface .
```
This is the wording I suggested in the Jira for the Upgrade Helper.

Reply:
sorry, lost the text earlier when I accepted the change in the GitHub UI, but then they were overwritten when I pushed another change from my working branch...

Reply:
Slightly modified the sentence order- 
If you want to reuse the self-hosted engine virtual machine to deploy the {virt-product-fullname} {engine-name} 4.4, note the MAC address of the self-hosted engine network interface before you shut it down.